index.html

```diff
@@ -6,19 +6,17 @@
     <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <section class=“img-container”>
-    <div id=left></div>
-    <div id=center></div>
-    <div id=right></div>
+    <p>do the thing</p>
+    <section id="img-container">
+      <img id="left">
```

Comment:
When we drew rows into our table, did we need to have tr tags hardcoded into our html to start with? Or did we just draw them all into a shared ancestor? How do you think that relates to these divs and the section tag around them?